src/helpers/sceneless-project.ts

```diff
@@ -2333,3 +2333,32 @@ export const createCompositor = async (
 
   return project
 }
+
+// In lieu of a versioning system, this can be used
+//  to update a ScenelessProject if its base structure changes.
+export const updateToLatest = async (_project: ScenelessProject) => {
```

Comment:
FYI, there's a series of ensure* functions in this file that are called when a project loads. If this function isn’t much different from those, I'd recommend moving it to that list to keep things uniform. This way, we won’t need to call it manually from Studio-next.

Reply:
I thought I already did this, but maybe it got deleted during some revision.